src/tests/general.edn

```diff
@@ -296,11 +296,6 @@
    (Count "tab11")
    (Assert.Is 2 true)
 
-   "My input"
-   (Dispatch inner1)
-   (Assert.Is "My input" true)
-   (Log)
-
```

Comment:
I would keep that sample and use `(| (Do inner1))` here. It may be a bit redundant with other tests, but looking at git history later in the future will clearly show the intent of the removal of Dispatch that way.

Reply:
Yeah might break coverage too maybe

Reply:
To me it doesn't make much sense to leave that as a test here if it's `(| (Do inner1))`, it wouldn't tests anything new.
If you wanted to figure out what happened to Dispatch, you'd just check the commit logs

Reply:
I added this PR comment to the commit message so people can find it